experiments/Cargo.toml

```diff
@@ -29,7 +29,10 @@ jexl-eval = "0.1.6"
 uuid = { version = "0.8", features = ["serde", "v4"]}
 sha2 = "0.9"
 hex = "0.4"
+uniffi = { git = "https://github.com/mozilla/uniffi-rs", rev = "686a8ec672cb6b86a6de920c43ff2d5678afd84a" }
 
+[build-dependencies]
+uniffi_build = { git = "https://github.com/mozilla/uniffi-rs", rev = "686a8ec672cb6b86a6de920c43ff2d5678afd84a", features=["builtin-bindgen"] }
```

Comment:
The `builtin-bindgen` here will cause cargo to build a local copy of the `uniffi-bindgen` tool and use that for generating the rust scaffolding.

An alternative is to have developers install `uniffi-bindgen` on their system, remove the `builtin-bindgen` feature, and let this use the system-provided tool. The potential advantage of this is that it's easier to ensure the kotlin bindings are generated using the same version of the tool; the downside is a less convenient developer experience.

Reply:
" the downside is a less convenient developer experience." is this just because they have to build it again? it didn't seem to take that long when I tried.

Reply:
The inconvenience is probably that developers have to ensure that they have `uniffi-bindgen` installed and in their `PATH` environment variable (a `cargo install` suffices here).
>  It's important that the Kotlin bindings are generated with the same version of uniffi as the rust scaffolding, and it's not obvious from this PR how you'll enforce that.

Ya I agree it's super important. When I add the kotlin bits I'll make sure we have a nice way to enforce that (Removing the `builtin-bindgen` seems like the most straight forward way)

Reply:
Right, yeah, it's an inconvenience in that it adds another thing to the list of prerequisites you have to install on your machine in order to build the project.